### ISSUE TITLE
Opening NC right menu from richdocuments is not possible if the user does not have sharing permission

### ISSUE THREAD

**Issue by maximelehericy** [nextcloud/richdocuments#3334]
**Describe the bug**
Usually in Collabora, the following icons are shown at the top right of the editor.
![image](https://github.com/nextcloud/richdocuments/assets/87175513/a84f2e4a-1417-41fa-8e69-2737dff1b4d7)

The following one allows to open the sharing tab of the nextcloud files, and from the sharing tab, you can access the details, or the comments, or the chat and video tab, which is quite useful if you want to comment or have a call at the same time you edit the document.
![image](https://github.com/nextcloud/richdocuments/assets/87175513/472a79c7-579b-45bc-9449-2afcb4eb6460)

Howewer, this icon is not present if the user cannot share the file. But even if the user cannot share the file, he might want to comment it, or have a call.
![image](https://github.com/nextcloud/richdocuments/assets/87175513/b80a86c6-a2b8-4279-adbf-9312c0cb1cc1)

**To Reproduce**
1. Create a folder (or a groupfolder) and an office document inside the folder
2. Share this folder with Alice, and remove the reshare permission
3. Log with Alice
4. Open the office document from the shared folder
5. Alice is not able to open the Nextcloud right menu
![image](https://github.com/nextcloud/richdocuments/assets/87175513/b80a86c6-a2b8-4279-adbf-9312c0cb1cc1)

**Expected behavior**
Even if the sharing permission is denied, Alice should be able to open the right side menu, as for any other document with sharing permission.

**Screenshots**
See above

**Client details:**
 - Device: [e.g. iPhone6, desktop]

## Server details
t.nc.c
**Operating system**:

**Web server:**

**Database:**

**PHP version:**

**Nextcloud version:** 27.1.2

**Version of the richdocuments app**

**Version of Collabora Online**

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```

</details>

**Comment by juliushaertl** [comment]
Yes, currently the sidebar button withing collabora is bound to the sharing option. This might be something to discuss with @pedropintosilva if we could separate that and have something like a file info button instead that we can show independently from sharing. This would also allow us to solve something that @schiessle recently mentioned to me that the talk sidebar is not accessible on public share links.
